### DISCUSSION TITLE
How to use root README?

### DISCUSSION THREAD

**Discussion by lars-reimann** [python-poetry/poetry#7617] [Q&A]
### Context

A project with the following abstract structure:

* `my_project/`
  * `pyproject.toml`
* `README.md`

Poetry version: 1.4.0

### Problem

When publishing to PyPI, I would like to use the root `README.md` (also shown on GitHub) to populate the [`Description`](https://packaging.python.org/en/latest/specifications/core-metadata/#description-optional). I would expect that I can simply include the root `README.md` in the `readme` field of `pyproject.toml`:

```toml
readme = "../README.md"
```

However, `poetry build` fails with a message

```
xy/README.md' is not in the subpath of 'xy/my_project' OR one path is relative and the other is absolute.
```

### Questions

1. How can I include the root `README`? Restructuring the repo is out of the question. Can this be done more elegantly than by copying the `README` manually or via a script?
2. Why is this restriction necessary? It seems like just the _contents_ of the `README` files are used and that their _location_ is irrelevant for the build output.

**Comment by samsja** [comment]
interested in the solution as well
